### PR TITLE
fix drop area bug

### PR DESCRIPTION
**Pull request by abolfazl-omidvarr**
fix a bug where drop area do not disappear when drag files from drop area into chat list on the left side of the app.

### CODE REVIEW

File: src/components/middle/composer/DropArea.tsx [hunk 1]
```diff
@@ -71,7 +71,10 @@ const DropArea: FC<OwnProps> = ({
     const { target: fromTarget, relatedTarget: toTarget } = e;
 
     // Esc button pressed during drag event
-    if ((fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget) {
+    if (
+      (fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget ||
```
Comment (Ajaxy) [inline src/components/middle/composer/DropArea.tsx:75]: Our ESLint rules do not allow mixing `&&` and `||`.

**Review by Ajaxy — CHANGES_REQUESTED** [review]
Thanks!

File: src/components/middle/composer/DropArea.tsx (hunk 1 above)
Comment (abolfazl-omidvarr) [reply src/components/middle/composer/DropArea.tsx]: hello thanks for your comment,
i edited the if condition in last commit.

is it still consider mixing || and  && in the edited condition?

File: src/components/middle/composer/DropArea.tsx [hunk 2]
```diff
@@ -71,7 +71,10 @@ const DropArea: FC<OwnProps> = ({
     const { target: fromTarget, relatedTarget: toTarget } = e;
 
     // Esc button pressed during drag event
-    if ((fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget) {
+    if (
+      ((fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && !toTarget) ||
+      !(toTarget as HTMLDivElement)?.matches('.DropTarget, .DropArea')
```
Comment (Ajaxy) [inline src/components/middle/composer/DropArea.tsx:76]: Did you maybe mean:

```suggestion
      fromTarget as HTMLDivElement).matches('.DropTarget, .DropArea') && (
        !toTarget || !(toTarget as HTMLDivElement)!.matches('.DropTarget, .DropArea')
      )
```

File: src/components/middle/composer/DropArea.tsx (hunk 1 above)
Comment (Ajaxy) [reply src/components/middle/composer/DropArea.tsx]: Thanks for your PR. You can check ESLint with `npm run check`.

Comment (abolfazl-omidvarr) [reply src/components/middle/composer/DropArea.tsx]: i committed to this branch according to your suggestion Alexander, your suggestion was a better approach.
thanks for your feedbacks

**Comment by Ajaxy** [comment]
Thank you! We will consider merging this in some near future.

**Pull request merged**